data/virt_autotest/host_unattended_installation_files/autoyast_opensuse_kvm_two_disk.xml

```diff
@@ -0,0 +1,151 @@
+<?xml version="1.0"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+  <general>
+    <mode>
+      <confirm t="boolean">false</confirm>
+      <final_reboot t="boolean">true</final_reboot>
+      <second_stage t="boolean">false</second_stage>
+    </mode>
+    <signature-handling>
+      <accept_file_without_checksum t="boolean">true</accept_file_without_checksum>
+      <accept_non_trusted_gpg_key t="boolean">true</accept_non_trusted_gpg_key>
+      <accept_unknown_gpg_key t="boolean">true</accept_unknown_gpg_key>
+      <accept_unsigned_file t="boolean">true</accept_unsigned_file>
+      <accept_verification_failed t="boolean">true</accept_verification_failed>
+      <import_gpg_key t="boolean">true</import_gpg_key>
+    </signature-handling>
+  </general>
+  <report>
+    <errors>
+      <show t="boolean">true</show>
+      <timeout t="integer">20</timeout>
+      <log t="boolean">true</log>
+    </errors>
+    <messages>
+      <show t="boolean">true</show>
+      <timeout t="integer">5</timeout>
+      <log t="boolean">true</log>
+    </messages>
+    <warnings>
+      <show t="boolean">true</show>
+      <timeout t="integer">10</timeout>
+      <log t="boolean">true</log>
+    </warnings>
+    <yesno_messages>
+      <show t="boolean">true</show>
+      <timeout t="integer">30</timeout>
+      <log t="boolean">true</log>
+    </yesno_messages>
+  </report>
+  <bootloader>
+    <loader_type>grub2</loader_type>
+    <global>
+      <activate>true</activate>
+      <boot_mbr>true</boot_mbr>
+      <gfxmode>auto</gfxmode>
+      <os_prober>false</os_prober>
+      <terminal>console serial</terminal>
+      <serial>serial --unit=1 --speed=115200 --word=8 --parity=no</serial>
+      <timeout t="integer">30</timeout>
+      <update_nvram>true</update_nvram>
+      <xen_kernel_append>console=com2,115200 loglvl=all guest_loglvl=all sync_console</xen_kernel_append>
+      <xen_append>console=hvc0,115200 console=tty loglevel=5</xen_append>
+    </global>
+  </bootloader>
+  <partitioning t="list">
+    <drive>
+      <device>/dev/sda</device>
+      <initialize t="boolean">true</initialize>
+      <use>all</use>
+      <enable_snapshots t="boolean">false</enable_snapshots>
+    </drive>
+    <drive>
+      <device>/dev/sdb</device>
```

Comment:
Is the main difference with one disk kvm template this section of /dev/sdb? What will be the solution if in future we get machines with eg 4 disks?

Reply:
Yes.

When we have machines with 3 or 4 disks and if the issue arises for Xen or KVM, we can work it under <partitioning> accordingly in the future, as of now not required.

The proposed solution involves creating two separate AutoYast profiles, one for machines with two or more hard disks and another for machines with a single disk. This way, we can ensure that tests for both scenarios work correctly. 

- This approach seems suitable for the specific testing and deployment scenarios. It offers a pragmatic solution to get tests running quickly and efficiently while acknowledging hardware differences.
- I believe it's important to document the purpose and usage of each profile clearly, both for the benefit of the current team and for anyone who may work with these profiles in the future.
- Regular testing and validation of these profiles on different hardware configurations will be essential to ensure that they continue to work as expected.
- As hardware configurations evolve, it may be necessary to revisit the profile strategy and consider whether a more scalable approach is needed.

In summary, while this solution may not be the most elegant from a purely technical perspective, it appears to be a practical and effective way to address the immediate need for testing on machines with different disk configurations as we need these tests to go into TW job group quickly. Careful documentation and ongoing testing will be key to its success.